### DISCUSSION TITLE
ESP32 board as an access point (AP)

### DISCUSSION THREAD

**Discussion by WolfOptics** [BIPES/BIPES#112] [Q&A] [answered]
Hi sirs,
 I want to use ESP32 as an access point (AP), and I also found EPS32 AP block in BIPES.
Do you have any example for this ? 

Best Regards,

**Comment by rafaelaroca** [comment, marked as answer]
Hello @WolfOptics. This is quite simple. Just add the AP block. Working example:

![image](https://user-images.githubusercontent.com/31751413/156560991-2841d2e9-26f9-456a-83d6-954872a28b6f.png)

This will create the micropython network with password 123.

After that, if you enable WebREPL, you will be able to use BIPES over wifi with your ESP32

regards
Rafael

**Reply by WolfOptics** [reply]
Thanks a lot. I got it.

**Comment by WolfOptics** [comment]
Thanks for your response. How can I show the IP address when I use ESP32 as
an AP?

Best Regards,

Rafael Vidal Aroca ***@***.***>於 2022年3月3日 週四，下午8:02寫道：

**Comment by rafaelaroca** [comment]
Hello, there is no block for it, but you can easily add a command inside a block and get the AP IP address:

Go to Python -> Run Python Code and type ap.ifconfig() on the block.

Here:

![image](https://user-images.githubusercontent.com/31751413/156579404-3b0e16bf-b4a7-48f1-b1bd-1a2aa954f3a3.png)

regards
Rafael

**Comment by WolfOptics** [comment]
Thanks a lot. Best Regards, 從 Windows 的郵件傳送 寄件者: Rafael Vidal Aroca傳送時間: 2022年3月3日 下午 10:00收件者: BIPES/BIPES副本: WolfOptics; Mention主旨: Re: [BIPES/BIPES] ESP32 board as an access point (AP) (Discussion #112) Hello, there is no block for it, but you can easily add a command inside a block and get the AP IP address:Go to Python -> Run Python Code and type ap.ifconfig() on the block.Here:regardsRafael—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you were mentioned.Message ID: ***@***.***>
